testapi.pm

```diff
@@ -1123,17 +1123,24 @@ sub type_string {
 
     my $max_interval = $args{max_interval}       // 250;
     my $wait         = $args{wait_screen_change} // 0;
-    bmwqemu::log_call(string => $log, max_interval => $max_interval, wait_screen_changes => $wait);
+    my $wait_still   = $args{wait_still_screen}  // 0;
```

Comment:
Please update the API description in the documentation above

Reply:
@okurz Good catch :)... /cc @yxususe